src/transformers/models/auto/image_processing_auto.py

```diff
@@ -106,6 +106,7 @@
         ("vit_hybrid", "ViTHybridImageProcessor"),
         ("vit_mae", "ViTImageProcessor"),
         ("vit_msn", "ViTImageProcessor"),
+        ("vitmatte", "VitMatteImageProcessor"),
```

Comment:
Would be nice to have a default for `vitedet` as well (even if it doesn't have its own image processor).

Reply:
VitDet was currently only added for `VitDetBackbone`. An image processor can theoretically be added once Mask R-CNN is added (#25348), which uses VitDet as backbone (and then use `MaskRCNNImageProcessor` as default image processor).